### DISCUSSION TITLE
How do I update the config of a tooltip instance ?

### DISCUSSION THREAD

**Discussion by zhnoah** [twbs/bootstrap#37360] [Q&A]
I've tried:
```js
const tooltip = Tooltip.getInstance('el');
tooltip._config.disabled = true;
```
But it didn't work.

**Comment by Edditoria** [comment]
Does `tooltip.disable()` what you need?
Is `tooltip` a returned result from `.getInstance('el')`?

**Reply by zhnoah** [reply]
disabled is just one example,
I wonder how to update the configuration of the instance.

**Comment by nkdas91** [comment]
Hi @zhnoah

The ```getInstance()``` function takes an **_HTML element_** as argument. But you have passed a **_String_**.
Also it needs to be ```bootstrap.Tooltip.getInstance()```.

Example:
```javascript
// Get the element with the Tooltip
const element = document.getElementById('example');

// Get or create Tooltip instance
const tooltip = bootstrap.Tooltip.getOrCreateInstance(element);

// Change content
tooltip.setContent({ '.tooltip-inner': 'New Content' })

// Disable
tooltip.disable();
```
